### ISSUE TITLE
Disable Native Sign-Up removes I forgot my password

### ISSUE THREAD

**Issue by GreenDragonTavern** [WWBN/AVideo#6672]
Client has closed system with 900 accounts he had me pre-load into AVideo. Everything works fine except when disabling Native Sign-Up you lose I forgot my password. Shouldn't work that way, Native Sign-Up has nothing to do with a current user's ability to recover their password.

**Comment by DanielnetoDotCom** [comment]
if the signup is not handling by AVideo, we cannot recover any password

**Comment by GreenDragonTavern** [comment]
All user data (as are all AVideo tables) reside in MariaDB, just like any normal installation. Only difference is instead of having 887 users create an account, I dumped them into the users table via SQL. I created a single dummy record to give me field data that is created using normal AVideo signup (exernalOptions field as an example) to use during bulk import. AVideo works fine, all is normal. This client DOES NOT WANT signups natively in AVideo (he has his own gig for subscribers and he feeds me data to import into AVideo). They log into AVideo, and password recovery should be available and not dependent on native signup option. I am just replacing the native signup with SQL but all else is the same.
